resources/windows_ad_domain_forest_2008.rb

```diff
@@ -0,0 +1,84 @@
+
+require 'mixlib/shellout'
+
+resource_name :windows_ad_domain_forest_2008
+
+provides :windows_ad_domain, platform: 'windows' do |node|
+  node['platform_version'].to_f < 6.2
+end
+
+property :safe_mode_pass, [String, nil], desired_state: false
+property :local_pass, [String, nil], desired_state: false
+property :options, Hash, default: {}, desired_state: false
+
+action :create do
+  if exists?
+    new_resource.updated_by_last_action(false)
```

Comment:
Instead of new_resource.updated_by_last_action you can just wrap the part that changes in a converge_by block, which allows you to add nice logging to the Chef run stating what the converge was.